packages/block-library/src/navigation-menu/edit.js

```diff
@@ -108,14 +113,70 @@ function NavigationMenu( {
 		}
 	};
 
-	useEffect( () => {
-		// Set/Unset colors CSS classes.
-		setAttributes( {
-			backgroundColorCSSClass: backgroundColor.class ? backgroundColor.class : null,
-			textColorCSSClass: textColor.class ? textColor.class : null,
-		} );
-	}, [ backgroundColor.class, textColor.class ] );
+	const handleCreateEmpty = () => {
+		setBlocksTemplate( null );
+		setInitialPlaceholder( false );
+	};
+
+	const handleCreateFromExisting = () => {
+		setBlocksTemplate( defaultPagesMenuItems );
+		setInitialPlaceholder( false );
+	};
+
+	//
+	// MARKUP
+	//
+
+	// Build Inline Styles
+	const navigationMenuInlineStyles = {
+		...( textColor && { color: textColor.color } ),
+		...( backgroundColor && { backgroundColor: backgroundColor.color } ),
+	};
+
+	// Build ClassNames
+	const navigationMenuClasses = classnames(
+		'wp-block-navigation-menu', {
+			'has-text-color': textColor.color,
+			'has-background-color': backgroundColor.color,
+			[ attributes.backgroundColorCSSClass ]: attributes && attributes.backgroundColorCSSClass,
+			[ attributes.textColorCSSClass ]: attributes && attributes.textColorCSSClass,
+		}
+	);
```

Comment:
These may be assigned after the placeholder return statement, for which they aren't needed.

Reply:
Good point. Fixed.